src/components/bulk-email-tool/bulk-email-form/bulk-email-recipient/BulkEmailRecipient.jsx

```diff
@@ -41,7 +42,7 @@ export default function BulkEmailRecipient(props) {
         <Form.Checkbox key="myself" value="myself" className="mt-2.5 col col-lg-4 col-sm-6 col-12">
           <FormattedMessage
             id="bulk.email.form.recipients.myself"
-            defaultMessage="Myself"
+            defaultMessage={RECIPIENTS_DISPLAY_NAMES.myself}
```

Comment:
I'm not sure but I think setting `defaultMessage` to a constant like this is going to break the extraction of strings for localization. In addition, it looks confusing because it looks like it's maybe a dynamic defaultMessage. Instead, all parts of this message descriptor (the `id`, the `defaultMessage`, and the `description`) should be put into a `messages` file and then you can just reference it here as `<FormattedMessage {...messages.recipientMyself} />`